pkg/generators/clientgen/gen.go

```diff
@@ -0,0 +1,476 @@
+/*
+Copyright The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clientgen
```

Comment:
Could we simplify the path and package name? e.g. `pkg/client/gen.go` or `pkg/generator/client/gen.go`

Just thinking about the import path; `github.com/kcp-dev/code-generator/pkg/generators/clientgen` vs. `github.com/kcp-dev/code-generator/pkg/client` (IMO, the latter has less stutter).

Reply:
👍 but maybe we can revisit overall package structure as a follow-up?

Reply:
Added this as a task in https://github.com/kcp-dev/code-generator/issues/4